docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1215,22 +1215,7 @@ function toggleBNR() {
     return;
   }
 
-  if (bnrEnabled) {
-    meeting.disableBNR().then((success) => {
-      if (success) {
-        bnrEnabled = false;
-        toggleBnrBtn.innerText = 'Enable BNR';
-      }
-    });
-  }
-  else {
-    meeting.enableBNR().then((success) => {
-      if (success) {
-        bnrEnabled = true;
-        toggleBnrBtn.innerText = 'Disable BNR';
-      }
-    });
-  }
+  console.log('BNR not supported');
```

Comment:
BNR will be provided by webrtc-core (once SPARK-399694 and SPARK-399695 are done)